contracts/FlowtyWrapped.cdc

```diff
@@ -260,6 +261,11 @@ pub contract FlowtyWrapped: NonFungibleToken, ViewResolver {
             return <- nft
         }
 
+        pub fun getEdition(editionName: String): AnyStruct{
+            let edition = FlowtyWrapped.getEditionRef(editionName)
+            return edition
+        }
```

Comment:
We will need to remove this, we cannot return a reference to an edition to the wild or anyone will be able to access them which could have unintended consequences

Reply:
Clearly I have been at it too long today, this is fine!